src/game-development.bib

```diff
@@ -0,0 +1,116 @@
+@article{blow2004game,
+  title={Game development: Harder than you think},
+  author={Blow, Jonathan},
+  journal={Queue},
+  volume={1},
+  number={10},
+  pages={28},
+  year={2004},
+  publisher={ACM}
+}
+
+@inproceedings{Anderson2008case,
+ author = {Anderson, Eike Falk and Engel, Steffen and Comninos, Peter and McLoughlin, Leigh},
+ title = {The Case for Research in Game Engine Architecture},
+ booktitle = {Proceedings of the 2008 Conference on Future Play: Research, Play, Share},
+ series = {Future Play '08},
+ year = {2008},
+ isbn = {978-1-60558-218-4},
+ location = {Toronto, Ontario, Canada},
+ pages = {228--231},
+ numpages = {4},
+ publisher = {ACM},
+ address = {New York, NY, USA},
+ keywords = {entertainment systems, game engine, game engine architecture},
+} 
+
+@inproceedings{washburn2016went,
+  title={What went right and what went wrong: an analysis of 155 postmortems from game development},
+  author={Washburn Jr, Michael and Sathiyanarayanan, Pavithra and Nagappan, Meiyappan and Zimmermann, Thomas and Bird, Christian},
+  booktitle={Proceedings of the 38th International Conference on Software Engineering Companion},
+  pages={280--289},
+  year={2016},
+  organization={ACM}
+}
+
+@inproceedings{elvezio2018mercury,
+  title={Mercury: A messaging framework for modular ui components},
+  author={Elvezio, Carmine and Sukan, Mengu and Feiner, Steven},
+  booktitle={Proceedings of the 2018 CHI Conference on Human Factors in Computing Systems},
+  pages={588},
+  year={2018},
+  organization={ACM}
+}
+
+@inproceedings{elliott1997functional,
+  title={Functional reactive animation},
+  author={Elliott, Conal and Hudak, Paul},
+  booktitle={ACM SIGPLAN Notices},
```

Comment:
don't cite SIGPLAN Notices (it just republishes conference papers) -- cite the original conference